contracts/Optimism_SpokePool.sol

```diff
@@ -44,6 +54,12 @@ contract Optimism_SpokePool is CrossDomainEnabled, SpokePoolInterface, SpokePool
     /**************************************
      *         DATA WORKER FUNCTIONS      *
      **************************************/
+
+    /**
+     * @notice Wraps any ETH into WETH before executing base function. This is neccessary because SpokePool receives
```

Comment:
is this correct? or is the point that you inherit the docs from the base spokePool for this method and simply append the information that differs to this comment?